pyccl/core.py

```diff
@@ -737,27 +738,65 @@ def __repr__(self):
 
     def compute_distances(self):
         """Compute the distance splines."""
+        if self.has_distances:
+            return
         status = 0
         status = lib.cosmology_compute_distances(self.cosmo, status)
         check(status, self)
 
     def compute_growth(self):
         """Compute the growth function."""
+        if self.has_growth:
+            return
+
+        if np.sum(self._params_init_kwargs['m_nu']) > 0:
```

Comment:
Is the (practical) limit for this really zero, or could we e.g. reasonable accurately compute it for the minimal mass?  That would seem likely to be a common use case.

Reply:
Yup. Using the number of massive neutrinos that CCL uses internally addresses this.